public/script.js

```diff
@@ -8513,14 +8513,17 @@ async function connectAPISlash(_, text) {
         $(apiConfig.button).trigger('click');
     }
 
-    toastr.info(`API set to ${text}, trying to connect..`);
+    const quiet = isTrueBoolean(args?.quiet);
+    quiet ? jQuery() : toastr.info(`API set to ${text}, trying to connect..`);
```

Comment:
Dare I ask why you decided to make that `jQuery()` call here?
Just to save two lines from an if block, and didn't want to do an inline if?